styles/styles.css

```diff
@@ -123,6 +120,65 @@ a:hover, a:active, a:focus, a {
     padding-top: 50px;
 }
 
+.artwork-container {
+	margin: 0 16px;
+}
+
+.artwork1_gallery_cell{
+	width: 100%;
+	height: 310px;
+	counter-increment: gallery-cell;
+	background: url("./../assets/hammond.svg") no-repeat center;
+	background-size: cover;
+}
+
+.artwork2_gallery_cell{
+	width: 100%;
+	height: 310px;
+	counter-increment: gallery-cell;
+	background: url("./../assets/hammond.svg") no-repeat center;
+	background-size: cover;
+}
+
+.artwork3_gallery_cell{
+	width: 100%;
+	height: 310px;
+	counter-increment: gallery-cell;
+	background: url("./../assets/hammond.svg") no-repeat center;
+	background-size: cover;
+}
+
+.artwork4_gallery_cell{
+	width: 100%;
+	height: 310px;
+	counter-increment: gallery-cell;
+	background: url("./../assets/hammond.svg") no-repeat center;
+	background-size: cover;
+}
+
+.artwork5_gallery_cell{
+	width: 100%;
+	height: 310px;
+	counter-increment: gallery-cell;
+	background: url("./../assets/hammond.svg") no-repeat center;
+	background-size: cover;
+}
+
+
+.artwork6_gallery_cell{
```

Comment:
you should create a class like "carousel-item" which has the css for width and height. and then remove the width and height from the gallery_cell.